tests/js/spec/components/forms/teamSelector.spec.jsx

```diff
@@ -26,16 +27,11 @@ const teamData = [
 ];
 const teams = teamData.map(data => TestStubs.Team(data));
 const project = TestStubs.Project({teams: [teams[0]]});
+const organization = TestStubs.Organization({access: ['project:write']});
```

Comment:
oh shoot thanks for this, looks much better

Reply:
needed to update it since it doesn't take those as props now :)